dts/arm/microchip/sam/sama7g5.dtsi

```diff
@@ -180,6 +180,13 @@
 			interrupts = <GIC_SPI 70 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
 		};
 
+		sha: sha@e1814000 {
```

Comment:
You are adding an entry in the devicetree were a binding does not exists yet.
Extract the binding from ead9f9fe in a separate commit. Put that new commit before this.